homeassistant/components/stookalert/binary_sensor.py

```diff
@@ -0,0 +1,88 @@
+"""This component provides support for Stookalert Binary Sensor."""
+from datetime import timedelta
+import logging
+
+import stookalert
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, STATE_OFF, STATE_ON
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+CONF_PROVINCE = "province"
+DEFAULT_NAME = "Stookalert"
+PROVINCES = [
+    "drenthe",
+    "flevoland",
+    "friesland",
+    "gelderland",
+    "groningen",
+    "limburg",
+    "noord-brabant",
+    "noord-holland",
+    "overijssel",
+    "utrecht",
+    "zeeland",
+    "zuid-holland",
```

Comment:
https://www.rivm.nl/stookalert is using the province names starting with a capital letter. To make it easier for the users we should use the same naming style as the source does.